CMakeLists.txt

```diff
@@ -91,8 +91,7 @@ install(
 )
 
 if(UNIX)
-  add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:-Wno-multichar;${BITS};${EXTRA_CXX_FLAGS}>")
-  add_compile_options("$<$<COMPILE_LANGUAGE:C>:${BITS}>")
+  add_compile_options("-Wno-multichar;-Wall")
 endif()
```

Comment:
On second thought, I think it's better to use `CMAKE_<LANG>_COMPILER_ID` check instead of platform check, like

```
if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
  add_compile_options(-Wno-multichar -Wall)
elseif()
```

Reply:
does "Clang" also cover "AppleClang" ?

Reply:
cf https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html

Reply:
Probably not, better to use `MATCHES` instead `STREQUAL` then.

Reply:
done